README.md

```diff
@@ -56,6 +56,20 @@ To run the hook over multiple files at the same time:
       files: ^requirements-dev\.(in|txt)$
 ```
 
+To check if the the lockfile is consistent with your dependencies (either with compiled requirements or toml):
+
+```yaml
+- repo: https://github.com/astral-sh/uv-pre-commit
+  # uv version.
+  rev: 0.3.0
```

Comment:
```suggestion
  rev: "0.3.1"
```
without quotes, this would be a float in YAML, so if we end up with `0.3.10` this gets converted to `0.3.1`, leading to the wrong version being used. Would be great to apply the same change in the other examples above.

Reply:
That won't be a float in YAML, at least in any conformant parser. Here's [the spec](https://yaml.org/spec/1.2.2/#1022-tag-resolution). It's only a problem for non-semver versions (where `3.10` python version is indeed interpreted as `3.1` float).

Reply:
Oh yeah I must be really tired, it obviously only affects cases with only `<major>.<minor>` :sweat_smile:

Reply:
a good read if either of you haven't seen before https://ruudvanasseldonk.com/2023/01/11/the-yaml-document-from-hell